include/libvfio-user.h

```diff
@@ -422,21 +461,22 @@ typedef struct {
 
 } vfu_migration_callbacks_t;
 
-typedef struct {
-    size_t                      size;
-    vfu_migration_callbacks_t   callbacks;
-    struct iovec                *mmap_areas;
-    uint32_t                    nr_mmap_areas;
-} vfu_migration_t;
-
-//TODO: Re-visit once migration support is done.
 /**
- * Enable support for device migration.
+ * vfu_setup_device_migration provides an abstraction over the migration
+ * protocol: the user specifies a set of callbacks which are called in respone
+ * to client accesses of the migration region; the migration read/write
```

Comment:
I think you mean the "region read/write callbacks are not called"; maybe add "after this function call".